app/src/routes/v2/template.js

```diff
@@ -6,12 +6,13 @@ const FileCache = require('../../components/fileCache');
 const { upload } = require('../../components/upload');
 const { truthy } = require('../../components/utils');
 const { middleware } = require('../../components/validation');
+const log = require('../../components/log')(module.filename);
 
 const fileCache = new FileCache();
 
 /** Returns the rendered report from cache */
 templateRouter.post('/', upload, async (req, res) => {
-  console.log('Template upload');
+  log.info('Template upload');
```

Comment:
Should we consider changing log level to verbose for all the log statements in this file? They're going to be mirrored by the http access log already and most of these messages are arguably not needed for general info/http level logging verbosity.